signac/contrib/project.py

```diff
@@ -301,7 +301,7 @@ def document(self):
 
         Returns
         -------
-        :class:`~signac.synced_collections.backends.collection_json.BufferedJSONAttrDict`
+        :class:`~signac._synced_collections.backends.collection_json.BufferedJSONAttrDict`
```

Comment:
Same as above: this class reference won't work because we're removing the synced collection docs. I wonder if we should add docs for the publicly imported name `signac.JSONDict` and point to that instead.